lib/widgets/course_detail_widgets.dart

```diff
@@ -21,7 +22,7 @@ class _GroupPickerState extends State<GroupPicker> {
     // which this would not detect
     for (int i = 0; i < widget._groups.length; ++i) {
       items.add(DropdownMenuItem(
-        child: Text((i + 1).toString()),
+        child: Text(widget._groups[i].getId()),
```

Comment:
Shouldn't we display both the group number and the ID? I know the id is encoded in the group but it's bad UX for new users who don't know, right?